### PR TITLE
Update README for Follow-Along Hardware

### PR DESCRIPTION
**Pull request by cdwilson**

### CODE REVIEW

**Review by ChrisGammell — COMMENTED** [review]
Please leave a link in the new section back to the project page ("For more details on this setup and photos of the result, check out the project page")

File: README.rst
Comment (cdwilson) [inline README.rst:372]: @cdwilson update this with the final link after https://github.com/golioth/projects/pull/40 is merged

File: README.rst
Comment (cdwilson) [inline README.rst:49]: @cdwilson fix this before merging

**Review by MarkoPura — APPROVED** [review]
I like it a lot! LGTM!
Let's wait for @szczys blessing before merging.

**Pull request merged**
